### PR TITLE
Updated adaptCobiGenTemplates function

### PR DESCRIPTION
**Pull request by dixyushi**

### CODE REVIEW

File: documentation/Functions.md [hunk 1]
```diff
@@ -394,11 +394,17 @@ The tutorials repository can be found https://github.com/devonfw-tutorials/tutor
 ***
 
 ### adaptTemplatesCobiGen <a name="adaptTemplatesCobiGen"></a>
+This function is used when you want to adapt CobiGen templates.
```
Comment (GuentherJulian) [inline documentation/Functions.md:397]: This function is used when you want to adapt CobiGen templates. See [CobiGen CLI documentation](https://devonfw.com/website/pages/docs/howto_Cobigen-CLI-generation.asciidoc.html).

File: documentation/Functions.md [hunk 2]
```diff
@@ -394,11 +394,17 @@ The tutorials repository can be found https://github.com/devonfw-tutorials/tutor
...
 #### parameter
-* No parameters
+* You don't need any parameters.
 #### example
 adaptTemplatesCobiGen()
 
+Note:
+1. No background script will run. Katacoda user will have to execute this command manually.
+
+2. To execute this command first you need to execute installCobiGen() function to install CobiGen. For this function CobiGen needs to be installed.
```
Comment (GuentherJulian) [inline documentation/Functions.md:406]: Mention only once that CobiGen must be installed.

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: documentation/Functions.md [hunk 3]
```diff
@@ -394,11 +394,19 @@ The tutorials repository can be found https://github.com/devonfw-tutorials/tutor
...
 #### parameter
-* No parameters
+* You don't need any parameters.
 #### example
 adaptTemplatesCobiGen()
 
+Note:
+1. No background script will run. 
+
+2. The command for execution will be generated by katacoda runner, so user will have to execute this command manually.
```
Comment (GuentherJulian) [inline documentation/Functions.md:406]: Katacoda

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: documentation/Functions.md (hunk 3 above)
Comment (dixyushi) [reply documentation/Functions.md:406]: done!!

**Pull request merged**